lib/fluent/plugin/in_http.rb

```diff
@@ -140,7 +140,7 @@ def close
       super
     end
 
-    def on_request(path_info, params)
+    def on_request(path_info, params, use_204_response=false)
```

Comment:
How do users set `use_204_response`?

Reply:
Oops, I think that is my misunderstanding.
I fixed it and add config_param in in_http.

This is my first time to PR with Ruby.
let me know if anything not suitable.
Thanks for review.